### PR TITLE
Find Route Added

### PR DESCRIPTION
**Pull request by rishub**
- Made home page look a little better
- Added Find Route to allow users to search for a ride
    - This is mainly under the search component which i added
    - Added [materialize-css](https://www.npmjs.com/package/materialize-css) and [react-materialize](https://www.github.com/react-materialize/react-materialize) packages in order to add the calendar component
         - materialize-css is global which sucks but it doesn't change too much
    - Used [react-select](https://www.github.com/JedWatson/react-select) component for selecting source and destination city
    - put possible destination cities in a config file under src/config/ so we can easily edit this

### CODE REVIEW

File: src/components/Search/index.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    // TODO: Check if all fields are complete and mabye push /rides/<params> ?
+    this.props.changePage("/rides");
+    // TODO: need to dispatch a fetch to the api here to get list of rides
+  };
+
+  render() {
+    return (
+      <div className="search">
+        <h3>Find a ride</h3>
+        <h5>
```
Comment (xuale) [inline src/components/Search/index.js:41]: Hopefully not anymore haha

**Review by RichardWen — COMMENTED** [review]
Are we moving Login into the landing page?

File: src/components/Search/index.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+          Currently we only support UCLA as an origin. Check back later for more
+          locations!
+        </h5>
+
+        <SelectInput
```
Comment (xuale) [inline src/components/Search/index.js:46]: Maybe change the landing page to a login/register page. Once logged in, the user can register a car, post a listing, and view/request available rides.

Comment (rishub) [reply src/components/Search/index.js:46]: yea i like that. we could either do that or keep the landing page as is and just let users try to find rides but redirect them to a login/register page if they're not logged in

**Review by xuale — COMMENTED** [review]
Looks good! Basically reached bpool v1. I'll work on making some of the components more reusable and restructuring some other stuff. I'll make a PR soon with my recommendations

**Pull request merged**